the missing lists (#605)
lsst-dm/dm_dev_guide: communications/mail.rst

#############
Mailing lists
#############

> Email is not the preferred mode of contact within DM but there are several project mailing list in use,

Reviewer: ```suggestion
Email is not the preferred mode of contact within DM but there are several project mailing lists in use.
```

> here is the full set of `Mailing lists`_. 

Reviewer: ```suggestion
The full set can be found at `Mailing lists`_. 
```

These lists are mainly populated by the Contacts Database groups.

> The relevant lists for DM are :

Reviewer: ```suggestion
The relevant lists for DM are:
```

> - dm-cams: Data Management CAM's

Reviewer: ```suggestion
- `dm-cams <https://lists.lsst.org/mailman/listinfo/dm-cams>`_: Data Management CAM's
```

> - dm-sst : DM System Science Team

Reviewer: ```suggestion
- `dm-sst <https://lists.lsst.org/mailman/listinfo/dm-sst>`_: DM System Science Team
```

> - dm-staff: All DM construction staff 

Reviewer: ```suggestion
- `dm-staff <https://lists.lsst.org/mailman/listinfo/dm-staff>`_: All DM construction staff 
```

> However these are seldom used and mentioned here for completeness.

Reviewer: ```suggestion
However, these are seldom used and mentioned here for completeness.
```
